codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithydotnet/TypeConversionCodegen.java

```diff
@@ -479,6 +479,24 @@ private String generateConstructorArg(final MemberShape memberShape) {
                 nameResolver.classPropertyForStructureMember(memberShape));
     }
 
+    // return false if this struct/member is one of the special ones with a IsXxxSet member
+    public boolean IsNormal(final MemberShape memberShape)
```

Comment:
Suggestion/ Issue: Normal is vague.
Smithy-Dafny has many edge cases, it helps to be specific.
This `Supports Is Set` would suggest a flip in the boolean returned,
but it better suggests what this method is for.
```suggestion
    public boolean memberSupportsIsSet(final MemberShape memberShape)
```